### PR TITLE
Rename the 'repository.config.xml' property to 'strongbox.config.xml'

### PR DESCRIPTION
**Pull request by IIEleniII**

### CODE REVIEW

**Comment by IIEleniII** [comment]
Hi Martin,

Thank you for the list^^ . Regarding my commits, do you want me to push them on a new  branch or should i continue pushing them on the master?

Kind regards, 

Eleni

**Comment by carlspring** [comment]
Doing it in your fork's master is fine.

**Comment by carlspring** [comment]
@IIEleniII ,

Any update?

**Comment by IIEleniII** [comment]
@carlspring  I changed the files in "Complete list", and  strongbox-web-integration-tests.

At the initial issue you mentioned running the code to see if all goes smoothly.Does it still holds?

**Comment by carlspring** [comment]
@cs-devops-bot add to whitelist

**Comment by carlspring** [comment]
@cs-devops-bot rebuild

**Comment by carlspring** [comment]
@IIEleniII ,

Thanks!

1. Would you mind granting me access to your fork?
2. Could you please have a look at https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md, sign the ICLA and send it back to me as described?

Many thanks!

Martin

File: strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java [hunk 1]
```diff
@@ -1,110 +0,0 @@
-package org.carlspring.strongbox.providers.repository;
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java]: What is going on here? Why has this test class been removed?

File: strongbox-commons/src/main/java/org/carlspring/strongbox/util/FileUtils.java [hunk 2]
```diff
@@ -1,38 +1,38 @@
-package org.carlspring.strongbox.util;
```
Comment (carlspring) [inline strongbox-commons/src/main/java/org/carlspring/strongbox/util/FileUtils.java]: Are there any actual changes to the file, apart from the change of the file permissions?

Comment (IIEleniII) [reply strongbox-commons/src/main/java/org/carlspring/strongbox/util/FileUtils.java]: No there aren't any.It wasn't on the list , and from what i saw it doen't need any changes.
I don't know why the file permissions have been changed

File: strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java (hunk 1 above)
Comment (IIEleniII) [reply strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java]: I haven't done any by hand changes in this file though...
Would it be possible that one of the changes in the other files can cause this?

Comment (carlspring) [reply strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java]: I don't know what you meant to do, but you will have to restore this file, which you seem to have accidentally deleted.

Comment (IIEleniII) [reply strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java]: Notepad++ .

**Comment by IIEleniII** [comment]
Regarding the fork, i  send you an invitation for collaborator in the repository.
Also i pulled a request for   strongbox-web-integration-tests .
Can you see these two actions?

File: strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java (hunk 1 above)
Comment (carlspring) [reply strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java]: I would recommend that you checkout our master and start fresh. Just copy the changed files and re-apply them one by one carefully. You can reset your fork by the number of commits that you have made and sync it with our and then re-apply your changes on top of it.

Comment (IIEleniII) [reply strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/LocalStorageProxyRepositoryExpiredArtifactsCleanerTestIT.java]: ok!i'll do that, thank you^^

**Comment by carlspring** [comment]
Yes, I can! Thanks!

I can only merge things in the strongbox-web-integration module once you have applied the changes to the strongbox project itself first.

**Comment by IIEleniII** [comment]
I opened a pull and mentioned you.

**Comment by carlspring** [comment]
Thanks!

**Pull request closed without merging**